adal/src/androidTest/java/com/microsoft/aad/adal/ChallengeResponseBuilderTests.java

```diff
@@ -306,14 +306,6 @@ public void testGetChallengeResponseInvalidRedirect() throws ClassNotFoundExcept
             assertTrue("Argument exception", ex.getCause().getMessage().contains("Nonce"));
         }
 
-        try {
-            m.invoke(handler, CERT_REDIRECT
-                    + "?Nonce=2&CertAuthoritiesMissing=ABC&Version=1.0&SubmitUrl=1&Context=1");
```

Comment:
Can you add a test where the `CertAuthorities` query param is returned in the same manner that originally uncovered this error? A test where it takes the form of `&CertAuthorities=` ? (empty list case)